backend/tests/codes/test_models.py

```diff
@@ -21,3 +21,68 @@ def test_code_factory():
     factories.CodeFactory.create_batch(10)
 
     assert Code.objects.count() == 10
+
+
+@pytest.mark.django_db
+def test_config_model():
+    config = Config(force_run = False, force_stop = False)
+    config.save()
+
+    assert Config.objects.count() == 1
+    assert config.force_run == False
+    
+
+@pytest.mark.django_db
+def test_config_model_save_multiple():
+    try:
+        factories.ConfigFactory.create_batch(3)
+        assert False
+    except:
+        assert True
+
+
+@pytest.mark.django_db
+def test_priority_queue_model():
+    example_code = factories.CodeFactory.create()
+    p_queue = AnimationPriorityQueueElement(code=example_code, priority=10)
+    p_queue.save()
+
+    returned_element = AnimationPriorityQueueElement.objects.order_by('-priority').first()
+    
+    assert AnimationPriorityQueueElement.objects.count() == 1
+    assert returned_element.priority == p_queue.priority
+
+
+@pytest.mark.django_db
+def test_priority_queue_model_mulitiple_objects():
+    factories.AnimationPriorityQueueElementFactory.create_batch(10)
+    
+    assert AnimationPriorityQueueElement.objects.count() == 10
+
+
```

Comment:
Warto napisać test, który robi to co jest w kolejce, czyli dodatkowo jeszcze usuwa pierwszy element z listy. 
Najlepiej napisać na to funkcję z usuwaniem i przetestować ją